test/indexer/test_hca_indexer.py

```diff
@@ -704,6 +703,42 @@ def test_metadata_generator(self):
             else:
                 self.assertIn(metadata_row['*.file_core.file_format'], ['fastq.gz', 'results', 'bam', 'bai'])
 
+    def test_metadata_field_exclusion(self):
+        self._index_canned_bundle(self.old_bundle)
+
+        # Check that the dynamic mapping has the field disabled
+        bundles_index = config.es_index_name('bundles')
+        mapping = self.es_client.indices.get_mapping(index=bundles_index)
+        self.assertFalse(mapping[bundles_index]['mappings']['doc']['properties']['metadata']['enabled'])
+
+        # Ensure that a metadata row exists …
+        hits = self._get_all_hits()
+        bundles_hit = one(hit['_source'] for hit in hits if hit['_index'] == bundles_index)
+        for metadata_row in bundles_hit['metadata']:
+            self.assertEqual(self.old_bundle, (metadata_row['bundle_uuid'], metadata_row['bundle_version']))
+
+        # … but that it can't be used for queries
+        empty_response = self.es_client.search(index=bundles_index,
```

Comment:
```suggestion
        hits = self.es_client.search(index=bundles_index,
```

Because A) it's not technically empty and B) `hits` is typically used in this module to refer to this type of value (an ES `search` response) and I find it more confusing to introduce a new variable name that's inconsistent with that convention. I also find reusing an existing variable name more expressive because it signals that you are done with the previous value.